### PR TITLE
modify files sort

### PR DESCRIPTION
**Pull request by zpfang**
solve the problem of the issue:
 The files sort may cause that the README.md file is after the child directory #8

test case as following:
Abc/README.md, isReadme: true
Abc/Abcd/README.md, isReadme: true
Abc/Abcd/Testd.md, isReadme: false
Abc/TEST.md, isReadme: false

Abc/README.md, isReadme: true 
this file is the dictionary, so it's should be the first one.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -36,10 +36,9 @@ module.exports = {
         },
         ( err, files ) => {
           files.sort((a, b) => {
-            const sameDir = path.dirname(a) == path.dirname(b)
-            if (sameDir && isReadme(a))
+            if (path.dirname(b).includes(path.dirname(a)) && isReadme(a))
```
Comment (julianxhokaxhiu) [inline index.js:39]: May I ask why did you remove the `const sameDir` declaration? Wouldn't have been easier to just patch that line with:

```js
const sameDir = path.dirname(b).includes(path.dirname(a))
```

Basically becomes a one-line patch.

**Review by julianxhokaxhiu — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (zpfang) [reply index.js:39]: I think about that it's having the two situations:

```
1. path.dirname(b).includes(path.dirname(a))
2. path.dirname(a).includes(path.dirname(b))
```
so I remove the const sameDir.

Comment (julianxhokaxhiu) [reply index.js:39]: Ah true, sorry. I did miss the swap of the letters. Visually they looked like the same.

Allright, safe enough to try :) Thank you!

**Review by julianxhokaxhiu — APPROVED** [review]
